tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -934,19 +975,25 @@ def test_get_colletion_with_dataset_no_metadata(self):
     def _test_response(self, collection_version: CollectionVersion, status_code=200, auth=True) -> dict:
         version_id = collection_version.version_id
         collection_id = collection_version.collection_id
+        is_open_revision = (
+            collection_version.published_at is None
+            and collection_version.canonical_collection.originally_published_at is not None
+        )
         headers = self.make_owner_header() if auth else self.make_not_owner_header()
         res_1 = self.app.get(f"/curation/v1/collections/{version_id}", headers=headers)
-        self.assertEqual(status_code, res_1.status_code)
-        if status_code == 200:
-            self.assertEqual(collection_version.collection_id.id, res_1.json["collection_id"])
+        if is_open_revision:
+            self.assertEqual(status_code, res_1.status_code)
+            if status_code == 200:
+                self.assertEqual(collection_version.collection_id.id, res_1.json["collection_id"])
```

Comment:
@nayib-jose-gloria I don't think this is correct. I must have missed this in the code. We should be returning the version id in this field for revisions.

Reply:
From the user's perspective, a revision id is a `collection_id`.

Reply:
it would simply be the same value. We had a discussion previously that implied this, I think. @brianraymor confirm?